newsletter/backend/newsletters/tests/test_models.py

```diff
@@ -0,0 +1,15 @@
+from newsletters.models import Agreements
+
+
+def test_representation_in_gui():
+    user_agreement = Agreements(email="test@newsletters.com")
+
+    assert str(user_agreement) == "test@newsletters.com"
+
+
+def test_newsletter_creation(newsletter):
```

Comment:
I would pass on this test, are You overriding save method? If not then we are not testing django code. 😉

Reply:
Erased this test. Also removed fixture from conftest.py.